source/_integrations/nordpool.markdown

```diff
@@ -164,6 +164,56 @@ data:
 
 {% endraw %}
 
+### Get price indices for date
+
+The integration can also provide price indices for any date with published prices. Use the "Get price indices for date" action to retrieve pricing information with a custom resolution time.
+
+The areas, currency, and resolution parameters are optional. If omitted, the values configured in the integration will be used and for resolution it will default to 60 minutes.
+
+{% configuration_basic %}
+Nord Pool configuration entry:
+  description: Select the Nord Pool configuration entry to target.
+Date:
+  description: Pick the date to fetch prices for (see note about possible dates below).
+Areas:
+  description: Select one market area to create output for. If omitted it will use the areas from the configuration entry.
+Currency:
+  description: Currency to display prices in. EUR is the base currency in Nord Pool prices.  If omitted it will use the currency from the configuration entry.
```

Comment:
```suggestion
  description: Currency to display prices in. EUR is the base currency in Nord Pool prices. If omitted, it will use the currency from the configuration entry.
```